tests/test_peer_review_orchestrator.py

```
@@ -0,0 +1,48 @@
+
+class PeerReviewOrchestratorTests(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.workflow = WORKFLOW.read_text(encoding="utf-8")
+        cls.policy = POLICY.read_text(encoding="utf-8")
+
+    def test_all_default_providers_have_documented_operator_commands(self) -> None:
+        self.assertIn("['coderabbit', '@coderabbitai full review']", self.workflow)
+        self.assertIn("['qodo', '/agentic_review']", self.workflow)
+        self.assertIn("['devin', '/devin review']", self.workflow)
+        self.assertIn("coderabbit:trigger_review,qodo:trigger_review,devin:trigger_review", self.workflow)
```

Pylint (4.0.6)

[convention] Line too long (106/100)

(C0301)